sources/background_scripts/notifications.js

```diff
@@ -0,0 +1,25 @@
+NewTabTweaker = {};
```

Comment:
Wrap the entire file in a closure `(function() { ... })();` as can be seen done in e.g. [image_context_menu.js](https://github.com/rharel/webext-new-tab-tweaker/blob/e55f7f48937ee2f36e2616edaffd73b29c2ea810/sources/background_scripts/image_context_menu.js#L1). This will make it easier to define module-local variables and functions without polluting the global namespace.